Tests/conf.json

```diff
@@ -20,6 +20,20 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "integrations": [
+                "WildFire-v2",
+                "Cortex XDR - IR",
+                "Joe Security"
+            ],
+            "instance_names": [
+                "WildFire-v2",
+                "Cortex XDR - IR",
+                "Joe_Security"
+            ],
+            "playbookID": "Test_Playbook_-_Cortex_XDR_Malware_-_Investigation_And_Response",
```

Comment:
"playbookID" should be the ID which should be the same as the name: `Test Playbook - Cortex XDR Malware - Investigation And Response`